src/components/account/AccountData.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import { View, Text } from 'react-native';
+import firebase from 'firebase';
+import { CardSection, Button, Title } from '../common';
+import { Size, FontWeight, Colors } from '../../constants/Styles';
+
+class AccountData extends Component {
+	state = {
+		email: ''
+	}
+
+	componentDidMount(){
+		this.setState({
+			email: this.verifyUser()
+		});
+	}
+
+	logOutButton = () => {
+		firebase.auth().signOut();
+		firebase.auth().onAuthStateChanged( user => {
+      if(!user) {
+				this.setState({
+					email: this.verifyUser()
+				});
+				this.props.navigation.navigate('storeList');
+      }
+		});
+	}
+
+	verifyUser = () => {
+		let user = firebase.auth().currentUser;
+		let email;
+		user != null ? email = user.email : email = 'Debes inicar sesión';
```

Comment:
change to `return !firebase.auth().currentUser ? 'Debes iniciar sesion : user.email'`